README.md

```diff
@@ -300,6 +300,8 @@ data-validation (--verbose or -v) validate custom-query
   --target-query-file TARGET_QUERY_FILE, -tqf TARGET_QUERY_FILE
```

Comment:
We should mention that 'The primary key should be included in the SELECT statement of both source_query.sql and target_query.sql' [here](https://github.com/GoogleCloudPlatform/professional-services-data-validator/blob/develop/README.md?plain=1#L280) too since this is the main documentation.